### PR TITLE
add aws sts docs

### PR DESCRIPTION
**Pull request by sallyom**
This adds the docs for setting up fulcio to use AWS STS OIDC identity provider

### CODE REVIEW

File: docs/sts/aws-sts.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+To continue, you need to have noted:
+
+- The OIDC Issuer URL for an IAM OIDC identity provider with Audience set to `sigstore`
+    - [AWS docs on creating OIDC identity provider](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc.html)
+- The ARN `arn:aws:iam::xxxx:role/xxxxxxxx` associated with the identity provider
+    - [AWS docs for creating roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user.html)
+
+### Configure Fulcio chart with AWS STS OIDC issuer
+
+After you have create teh IAM OICD identity provider with an associated IAM role, update the fulcio server configuration.
```
Comment (ritz303) [inline docs/sts/aws-sts.md:23]: Typo: `teh` --> `the`

Comment (sallyom) [reply docs/sts/aws-sts.md]: updated, thanks

**Comment by sallyom** [comment]
@lance ptal, thanks! 
@sabre1041 I brought suggestions from your feedback from the [previous PR](https://github.com/securesign/sigstore-ocp/pull/26) here as well.

File: docs/sts/aws-sts.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    - [AWS docs on creating OIDC identity provider](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc.html)
+- The ARN of a role for web identity federation `arn:aws:iam::xxxx:role/xxxxxxxx` associated with the identity provider
+    - [AWS docs for creating roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user.html)
+
+### Configure Fulcio chart with AWS STS OIDC issuer
+
+After you have created the IAM OIDC identity provider with an associated IAM role, update or configure the fulcio server configuration.
+The configuration is stored in a configmap named `fulcio-server-config` in the same namespace where the `fulcio-server` is running..
+
+Update fulcio-server-config to add the AWS OIDC identity provider issuer URL:
```
Comment (sallyom) [inline docs/sts/aws-sts.md:26]: TODO: document how to with the SecureSign Custom Resource

Comment (sallyom) [reply docs/sts/aws-sts.md:30]: (in follow-up PR)

**Comment by sallyom** [comment]
@cooktheryan ptal

**Comment by cooktheryan** [comment]
@ritz303 I believe this is ready for review

**Comment by cooktheryan** [comment]
this looks good from my side waiting on @ritz303 for  a +1

**Comment by ritz303** [comment]
@sallyom : Sorry, for the delay here :-( !  I guess I missed the notification to review this.  I'll take a look.

File: docs/sts/aws-sts.md
Comment (ritz303) [inline docs/sts/aws-sts.md:1]: LGTM

**Pull request merged**